dataloader.py

```diff
@@ -255,9 +257,8 @@ class DnaSequenceMapper:
 
     def __init__(self):
         nucleobase_symbols = ["A", "C", "G", "T", "N"]
-        padding_character = [" "]
 
-        self.nucleobase_letters = sorted(nucleobase_symbols + padding_character)
+        self.nucleobase_letters = sorted(nucleobase_symbols)
 
         self.num_nucleobase_letters = len(self.nucleobase_letters)
```

Comment:
Not an excellent way and feels somewhat hacky, but in another project I simply saved the variable in the configuration object which is afterwards used to create the network instance:

https://github.com/Ensembl/gene_pcp/blob/main/utils.py#L247-L249

https://github.com/Ensembl/gene_pcp/blob/main/transformer_pipeline.py#L448-L456

https://github.com/Ensembl/gene_pcp/blob/main/transformer_pipeline.py#L217

I think it will work for this project as well.